.github/workflows/callable_test-automation.yaml

```
@@ -261,11 +257,11 @@ jobs:
           key: ${{ runner.os }}-buildx-${{ github.sha }}
           restore-keys: |
             ${{ runner.os }}-buildx
-      - name: Download Pipeline sovrin-deb artifact
-        uses: actions/download-artifact@v3
-        with:
-          name: sovrin-deb
-          path: ./system/docker/node/
+     - name: Download Pipeline sovrin.deb artifact
+       uses: actions/download-artifact@v3
+       with:
+         name: sovrin.deb
+         path: ./system/docker/node/
```

```suggestion
     - name: Download Pipeline sovrin-deb artifact
       uses: actions/download-artifact@v3
       with:
         name: sovrin-deb
         path: ./system/docker/node/
```
Otherwise the download of the artifact won't work.